### PR TITLE
Tables feedback

### PR DESCRIPTION
**Pull request by apothuku**
Each piece of feedback addressed can be found in the commit messages of this PR

### CODE REVIEW

File: cash-bail-and-race.html [hunk 1]
```diff
@@ -36,12 +36,10 @@
               <div class="default text">Search counties</div>
               <div class="menu"></div>
             </div>
-            <div class="btn-text outliers-btn">Show Outliers</div>
           </div>
           <div class="switch-container">
-            <div class="btn-text switch left showing">% Cash Bail</div>
-            <div class="btn-text switch right">Bail Amount</div>
-            <div class="btn-text outliers-btn mobile-only">Show Outliers</div>
+            <div class="btn-text switch left showing">Cash bail rt.</div>
```
Comment (alexwgraves) [inline cash-bail-and-race.html:41]: This is inconsistent with `bail-trends`, which is "Cash Bail" (capitalized, no "rt."). We're okay with that?

**Review by alexwgraves — APPROVED** [review]
One small question (which might be for Rong) but other than that LGTM!

File: src/css/table.css [hunk 2]
```diff
@@ -17,13 +17,16 @@ tr.expanded {
   background-color: #131313;
 }
 
+tr.summary-row {
```
Comment (marioaks) [inline src/css/table.css:20]: Not a big deal but let's not forget to change this to `.table-container > table tr.sumary-row` when we merge our branches

**Review by marioaks — APPROVED** [review]
Looks good

**Pull request merged**